tests/core/audit_test.py

```diff
@@ -441,15 +442,7 @@ def test_input_message(self, prompt_secret_decision, expected_output):
 
 @pytest.fixture
 def mock_printer():
```

Comment:
Super nit: Maybe consider renaming to avoid the name conflict/alias of the `mock_printer` import, e.g. `mock_audit_printer` or something.